libraries/AP_Parachute/AP_Parachute.cpp

```diff
@@ -59,6 +59,23 @@ const AP_Param::GroupInfo AP_Parachute::var_info[] = {
     // @Increment: 1
     // @User: Standard
     AP_GROUPINFO("DELAY_MS", 5, AP_Parachute, _delay_ms, AP_PARACHUTE_RELEASE_DELAY_MS),
+
+    // @Param: EMERGENCY
+    // @DisplayName: Emergency parachute enable
+    // @Description: Releases the parachute when an emergency occurs
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO("EMERGENCY", 6, AP_Parachute, _emergency_enabled, AP_PARACHUTE_EMERGENCY_DEFAULT),
+    
+    // @Param: MAX_SINK
+    // @DisplayName: Maximum sink speed in m/s
```

Comment:
The display name should mention parachute somewhere, otherwise this will look like it's possibly a vehicle max sink rate to some users.

Reply:
Agreed, i think we can make it better renaming the parameter to CRITICAL_SINK. It sounds more like an emergency.